### PR TITLE
Add addon upgrade plan instructions

### PR DESCRIPTION
**Pull request by MaximilianMeister**
Signed-off-by: Maximilian Meister <mmeister@suse.de>

### CODE REVIEW

File: adoc/admin-updates.adoc [hunk 1]
```diff
@@ -94,6 +94,40 @@ Latest Kubernetes version: 1.15.0
...
+In order to get an overview of the addon updates available, you can run:
+
+----
+skuba addon upgrade plan
+----
+
+This will show you a list of updates (if available) for different addons
+installed on the cluster:
+
+---
```
Comment (jenting) [inline adoc/admin-updates.adoc]: the format should be `----`, not `---`

File: adoc/admin-updates.adoc [hunk 2]
```diff
@@ -94,6 +94,40 @@ Latest Kubernetes version: 1.15.0
...
+----
+skuba version: 1.0.0 (release) 20190927 go1.12.9
+Current Kubernetes cluster version: 1.15.2
+Latest Kubernetes version: 1.15.2
+
+Congratulations! Addons are already at the latest version available
+----
 
 [TIP]
 =====
```
Comment (jenting) [inline adoc/admin-updates.adoc]: recommend moving this TIP under _Platform Updates_

File: adoc/admin-updates.adoc (hunk 1 above)
Comment (MaximilianMeister) [reply adoc/admin-updates.adoc]: updated thanks

**Comment by nkoranova** [comment]
Looks good, thanks for the PR!

**Pull request merged**